--- patches/evhelper.cpp.orig	2026-07-22 05:11:21.692264345 +0000
+++ pvxs/src/evhelper.cpp	2026-07-22 05:11:29.384298458 +0000
@@ -175,17 +175,18 @@
         try {
             evconfig conf(__FILE__, __LINE__, event_config_new());
 #ifdef __rtems__
             /* with libbsd circa RTEMS 5.1
              * TCP peer close/reset notifications appear to be lost.
              * Maybe due to absence of NOTE_EOF?
              * poll() seems to work though.
              */
-            event_config_avoid_method(conf.get(), "kqueue");
+            /* rtems-cside panel: disabled to test kqueue on RTEMS 6 */
+            /* event_config_avoid_method(conf.get(), "kqueue"); */
 #endif
             decltype (base) tbase(__FILE__, __LINE__, event_base_new_with_config(conf.get()));
             if(evthread_make_base_notifiable(tbase.get())) {
                 throw std::runtime_error("evthread_make_base_notifiable");
             }
 
             evevent handle(__FILE__, __LINE__,
                            event_new(tbase.get(), -1, EV_TIMEOUT, &doWorkS, this));
